.github/workflows/python-package.yml

```diff
@@ -27,18 +27,17 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip wheel
-        pip install -e .[dev]
+        python -m pip install --upgrade pip wheel poetry
```

Comment:
Question: this is not the official way to install poetry normally (which is stupidly done with curl) is this an actions-specific way?

Reply:
I did not know that, thanks! It just seemed like the simplest, most cross-platform way to install poetry. Looking at the docs now, it looks like you would need different commands to install it for Unix vs Windows. Looks like there are a couple of options for doing it in the actions marketplace so I'll give that a try.

Reply:
Might not be the recomended way from a developers perspective who might have different pythin versions, etc. But for Actions, which is a 1-off use of each platform and python version, I think using pip is the simplest approach.

Reply:
I've just substituted [this](https://github.com/abatilo/actions-poetry) (which actually just does exactly the same thing with `pip`, but is a bit neater and allows you to specify a Poetry version in Actions-style).